edu/jalc/automobile/onlinebuilder/components/suspension/economy/EconomySuspension.java

```diff
@@ -49,6 +49,7 @@ public String toString(){
       return "EconomySuspension has :\n" +
               "\t"+ stockShock.toString()+"\n"+
               "\t"+ stockSpring.toString()+"\n"+
+              "\t"+ wheel.toString() +"\n" +
```

Comment:
Threw my code off for hours due to this not being implemented in this already haha.